predicates/traffic/traffic.go

```diff
@@ -0,0 +1,140 @@
+/*
+Package traffic implements a custom predicate to control
+traffic by setting rates for certain routes.
+
+The chance for hitting a route is defined by the mandatory first
+parameter, that must be a number between 0 and 1.
+
+The optional second argument is used to specify the cookie
+name for the traffic group.
```

Comment:
I'd also add a short explanation. "Stickiness allows all subsequent requests form the same client to match the same route"